src/components/pages/Landing/landingPageStyledComponents.js

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components';
+

```

Comment:
should these changes outside the readme go with a different PR?

Reply:
They should! I was a little bit confused about how to push that code. It came in when I did a pull request to make sure that the branched stayed current. I need to go about removing all of the information from this commit not related to the updated readme.